vllm/model_executor/layers/quantization/modelopt.py

```diff
@@ -30,6 +37,14 @@
 from vllm.platforms import current_platform
 from vllm.scalar_type import scalar_types
 
+try:
+    from flashinfer import fp4_quantize as fp4_quantize
+    from flashinfer.fused_moe import (
+        cutlass_fused_moe as flashinfer_cutlass_fused_moe)
+except ImportError:
+    if not TYPE_CHECKING:
+        flashinfer_cutlass_fused_moe = None
```

Comment:
We should make a common wrapper for this. I can work on a wrapper similar to vllm/utils/deep_gemm.py